package.json

```diff
@@ -51,7 +51,7 @@
 		"!dist/**/tests"
 	],
 	"peerDependencies": {
-		"svelte": "^3.58.0"
+		"svelte": "^4.1.1"
```

Comment:
Actually, I wonder if instead we should do `"svelte": ">=3 <5"`

I guess it depends if this change would still make it work with Svelte 3 🤔

Reply:
I thought about it, but with Svelte 3, there are certain type errors. As such, I think peer dependencies should mark the versions with which the package works _flawlessly_, and if people are willing to put up with type errors, they're probably gonna be willing to put up with peer dependency errors

Reply:
Makes sense!